src/byui/cit260/sevenyearsofplenty/control/CropsControl.java

```diff
@@ -121,4 +121,31 @@ public static int harvestCrops(Crops theCropsObj) {
     } 
     
     
+
+        //reqNutrition is how much each Member of the Population needs each year
+    //to be sustained.  
+    public static int feedPeople(int reqNutrition, int feedAmount, Crops theCropsObj) {
```

Comment:
Is reqNutrition a variable calculated elsewhere? Where is this number being decided?